src/hooks/viewport/useMediaQuery.tsx

```diff
@@ -1,11 +1,11 @@
 import { useCallback, useEffect, useState } from "react";
 
-type MediaType = "isMobile" | "isTablet" | "isLaptop" | "isLaptopMin";
+type MediaType = "isMobile" | "isTablet" | "isLaptop" | "isLaptopSmall";
 
 const mediaBreakpoints: Record<MediaType, string> = {
   isMobile: "(max-width: 320px)",
-  isTablet: "(max-width: 768px)",
-  isLaptopMin: "(max-width: 960px)",
+  isTablet: "(max-width: 767px)",
+  isLaptopSmall: "(max-width: 960px)",
   isLaptop: "(max-width: 1024px)",
```

Comment:
[NOTE] For some future PR to extract these values in a `src/settings/` file just like they ar in a `src/scss` one

Reply:
[ce03ad0](https://github.com/sparkletown/sparkle/pull/2973/commits/ce03ad0c063b92163c5039c84f70f34635278d0b)